JitsiMeetJS.js

```diff
@@ -197,7 +197,11 @@ var LibJitsiMeet = {
               handler(error);
           });
         } else {
-            Statistics.sendUnhandledError(error);
+            if (error instanceof JitsiTrackError && error.gum) {
+                Statistics.sendGetUserMediaFailed(error);
+            } else {
+                Statistics.sendUnhandledError(error);
+            }
```

Comment:
This piece looks suspiciously close to a duplicate of the piece in JitsiConference.js. Is it possible to eliminate this duplication?